caikit/interfaces/nlp/tasks.py

```diff
@@ -32,3 +40,41 @@ class TextGenerationTask(TaskBase):
     """The Text Generation Task is responsible for taking input prompting text
     and generating additional text from that prompt.
     """
+
+
+@task(
+    required_parameters={"text": str},
+    output_type=ClassificationResult,
+)
+class TextClassificationTask(TaskBase):
```

Comment:
I think `task` definitions should probably have docstrings to describe the semantics of what the task does. It would probably be good to have docstrings for the dataobjects too, but tasks seem the most important to me